content/blogposts/2024/improving-cody-autocomplete-faster-smarter.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Improving Cody Autocomplete: Faster and Smarter"
+authors:
+  - name: Chris Sev
+    url: https://twitter.com/chris__sev
+publishDate: 2024-08-21T10:00-01:00
+description: "We're excited to announce a new release of Cody that includes faster autocomplete and smarter code suggestions."
+tags: [blog]
+slug: "improving-cody-autocomplete-faster-smarter"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/faster-autocomplete/autocomplete-cover.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/faster-autocomplete/autocomplete-cover.png
+--- 
+
+We just wrapped up our Faster Autocomplete project and now Cody is faster and smarter for more of your coding tasks. The big changes that happened under the hood are that we switched from Starcoder to DeepSeek v2 for our autocomplete model leading to improvements across the board.
```

Comment:
I believe these should be formatted as "StarCoder" and "DeepSeek-V2" (based on huggingface)

```suggestion
We just wrapped up our Faster Autocomplete project and now Cody is faster and smarter for more of your coding tasks. The big changes that happened under the hood are that we switched from StarCoder to DeepSeek-V2 for our autocomplete model leading to improvements across the board.
```